Source/loadsave.cpp

```diff
@@ -150,13 +156,19 @@ class SaveHelper {
 	{
 	}
 
-	bool IsValid(uint32_t len = 1)
+	bool IsValid(size_t len = 1)
 	{
 		return m_buffer_ != nullptr
 		    && m_capacity_ >= (m_cur_ + len);
 	}
 
-	void Skip(uint32_t len)
+	template<typename T>
+	constexpr void Skip(void)

```

Comment:
The `void` is unnecessary in C++

```suggestion
	constexpr void Skip()
```

(same above in LoadHelper)